src/utils/FetcherConfiguration.ts

```diff
@@ -1,5 +1,6 @@
-export type RequestOptions = {
-  url: string
+export type RequestOptions = Partial<CompleteRequestOptions>
+
+export type CompleteRequestOptions = {
```

Comment:
Just want to make sure that it was intentional to leave `body` and `headers` as optional :)

Reply:
Yes, I left them as mandatory as they are not used in all requests. Do you have any other idea?